### ISSUE TITLE
Java 12 error

### ISSUE THREAD

**Issue by andris155** [SBPrime/AsyncWorldEdit#223]
```
[19:52:01] [Server thread/INFO]: [AsyncWorldEdit] Unable to invoke findLoadedClass: unsupported version.
[19:52:01] [Server thread/INFO]: [AsyncWorldEdit] ***********************************
[19:52:01] [Server thread/INFO]: [AsyncWorldEdit] *
[19:52:01] [Server thread/INFO]: [AsyncWorldEdit] * Exception: java.lang.NoSuchFieldException
[19:52:01] [Server thread/INFO]: [AsyncWorldEdit] * Error message: modifiers
[19:52:01] [Server thread/INFO]: [AsyncWorldEdit] * Stack: 
[19:52:01] [Server thread/INFO]: [AsyncWorldEdit] * java.base/java.lang.Class.getDeclaredField(Class.java:2417)
[19:52:01] [Server thread/INFO]: [AsyncWorldEdit] * org.primesoft.asyncworldedit.utils.Reflection.invoke(Reflection.java:118)
[19:52:01] [Server thread/INFO]: [AsyncWorldEdit] * org.primesoft.asyncworldedit.Loader.findLoadedClass(Loader.java:360)
[19:52:01] [Server thread/INFO]: [AsyncWorldEdit] * org.primesoft.asyncworldedit.Loader.loadClass(Loader.java:189)
[19:52:01] [Server thread/INFO]: [AsyncWorldEdit] * java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
[19:52:01] [Server thread/INFO]: [AsyncWorldEdit] * org.primesoft.asyncworldedit.AsyncWorldEditBukkit.createInjector(AsyncWorldEditBukkit.java:327)
[19:52:01] [Server thread/INFO]: [AsyncWorldEdit] * org.primesoft.asyncworldedit.AsyncWorldEditBukkit.inject(AsyncWorldEditBukkit.java:231)
[19:52:01] [Server thread/INFO]: [AsyncWorldEdit] * org.primesoft.asyncworldedit.AsyncWorldEditBukkit.<clinit>(AsyncWorldEditBukkit.java:96)
[19:52:01] [Server thread/INFO]: [AsyncWorldEdit] * java.base/java.lang.Class.forName0(Native Method)
[19:52:01] [Server thread/INFO]: [AsyncWorldEdit] * java.base/java.lang.Class.forName(Class.java:415)
[19:52:01] [Server thread/INFO]: [AsyncWorldEdit] * org.bukkit.plugin.java.PluginClassLoader.<init>(PluginClassLoader.java:64)
[19:52:01] [Server thread/INFO]: [AsyncWorldEdit] * org.bukkit.plugin.java.JavaPluginLoader.loadPlugin(JavaPluginLoader.java:134)
[19:52:01] [Server thread/INFO]: [AsyncWorldEdit] * org.bukkit.plugin.SimplePluginManager.loadPlugin(SimplePluginManager.java:334)
[19:52:01] [Server thread/INFO]: [AsyncWorldEdit] * org.bukkit.plugin.SimplePluginManager.loadPlugins(SimplePluginManager.java:255)
[19:52:01] [Server thread/INFO]: [AsyncWorldEdit] * org.bukkit.craftbukkit.v1_13_R2.CraftServer.loadPlugins(CraftServer.java:331)
[19:52:01] [Server thread/INFO]: [AsyncWorldEdit] * net.minecraft.server.v1_13_R2.DedicatedServer.init(DedicatedServer.java:235)
[19:52:01] [Server thread/INFO]: [AsyncWorldEdit] * net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:787)
[19:52:01] [Server thread/INFO]: [AsyncWorldEdit] * java.base/java.lang.Thread.run(Thread.java:835)
[19:52:01] [Server thread/INFO]: [AsyncWorldEdit] ***********************************
[19:52:01] [Server thread/INFO]: [AsyncWorldEdit] ***********************************
[19:52:01] [Server thread/INFO]: [AsyncWorldEdit] Unable to create AWE Injector: unsupported version.
[19:52:01] [Server thread/INFO]: [AsyncWorldEdit] ***********************************
[19:52:01] [Server thread/INFO]: [AsyncWorldEdit] *
[19:52:01] [Server thread/INFO]: [AsyncWorldEdit] * Exception: java.lang.NoSuchFieldException
[19:52:01] [Server thread/INFO]: [AsyncWorldEdit] * Error message: modifiers
[19:52:01] [Server thread/INFO]: [AsyncWorldEdit] * Stack: 
[19:52:01] [Server thread/INFO]: [AsyncWorldEdit] * java.base/java.lang.Class.getDeclaredField(Class.java:2417)
[19:52:01] [Server thread/INFO]: [AsyncWorldEdit] * org.primesoft.asyncworldedit.utils.Reflection.create(Reflection.java:81)
[19:52:01] [Server thread/INFO]: [AsyncWorldEdit] * org.primesoft.asyncworldedit.AsyncWorldEditBukkit.createInjector(AsyncWorldEditBukkit.java:335)
[19:52:01] [Server thread/INFO]: [AsyncWorldEdit] * org.primesoft.asyncworldedit.AsyncWorldEditBukkit.inject(AsyncWorldEditBukkit.java:231)
[19:52:01] [Server thread/INFO]: [AsyncWorldEdit] * org.primesoft.asyncworldedit.AsyncWorldEditBukkit.<clinit>(AsyncWorldEditBukkit.java:96)
[19:52:01] [Server thread/INFO]: [AsyncWorldEdit] * java.base/java.lang.Class.forName0(Native Method)
[19:52:01] [Server thread/INFO]: [AsyncWorldEdit] * java.base/java.lang.Class.forName(Class.java:415)
[19:52:01] [Server thread/INFO]: [AsyncWorldEdit] * org.bukkit.plugin.java.PluginClassLoader.<init>(PluginClassLoader.java:64)
[19:52:01] [Server thread/INFO]: [AsyncWorldEdit] * org.bukkit.plugin.java.JavaPluginLoader.loadPlugin(JavaPluginLoader.java:134)
[19:52:01] [Server thread/INFO]: [AsyncWorldEdit] * org.bukkit.plugin.SimplePluginManager.loadPlugin(SimplePluginManager.java:334)
[19:52:01] [Server thread/INFO]: [AsyncWorldEdit] * org.bukkit.plugin.SimplePluginManager.loadPlugins(SimplePluginManager.java:255)
[19:52:01] [Server thread/INFO]: [AsyncWorldEdit] * org.bukkit.craftbukkit.v1_13_R2.CraftServer.loadPlugins(CraftServer.java:331)
[19:52:01] [Server thread/INFO]: [AsyncWorldEdit] * net.minecraft.server.v1_13_R2.DedicatedServer.init(DedicatedServer.java:235)
[19:52:01] [Server thread/INFO]: [AsyncWorldEdit] * net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:787)
[19:52:01] [Server thread/INFO]: [AsyncWorldEdit] * java.base/java.lang.Thread.run(Thread.java:835)
[19:52:01] [Server thread/INFO]: [AsyncWorldEdit] ***********************************
[19:52:01] [Server thread/INFO]: [AsyncWorldEdit] ERROR: Injector not found.
```

**Comment by SBPrime** [comment]
Look like a field is not available in the class. Although when I checked the source of open jdk the field is available. What version of java are you using?

**Comment by andris155** [comment]
https://www.oracle.com/technetwork/java/javase/downloads/jdk12-downloads-5295953.html

**Comment by SBPrime** [comment]
Again the modifiers field is present in the source of the JDK (found in java.base). Please what exact libs are you using when starting java, what is the exact version.

**Comment by andris155** [comment]
I used jdk-12.0.1_linux-x64_bin.tar.gz

**Comment by SBPrime** [comment]
Thats the one that I checked. Please veriffy that it loads the java.base modules.

**Comment by BOOM21** [comment]
As mentioned in my falsely created Issue I suffer from the same problem. As explained there, I made sure all needed modules are privided.

**Comment by Rincewind34** [comment]
Any news on this one? I experience the exception too. Could it probably be an illegal reflective access?

**Comment by Rincewind34** [comment]
Only java 1.8 is supported.
Duplicate of https://github.com/SBPrime/AsyncWorldEdit/issues/223

_Originally posted by @SBPrime in https://github.com/SBPrime/AsyncWorldEdit-Premium/issues/180#issuecomment-500207434_

Well nothing more to be done here...

**Comment by andris155** [comment]
Works perfectly on Java 11.

**Comment by Rincewind34** [comment]
Why don't you use FAWE instead of this plugin, when you're already running FAVS?

**Comment by SBPrime** [comment]
Partial work started in 3.6.11, support for java 12 should be available in 3.7.0 (most likely next release).

**Comment by andris155** [comment]
The updated version of Voxel works perfectly with AsyncWorldEdit if you compile with Java 11.
